sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3831,6 +3831,15 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val PIPE_OPERATOR_ALLOW_AGGREGATE_IN_SELECT =
+    buildConf("spark.sql.pipeOperator.allowAggregateInSelect")
```

Comment:
Do we need to add this new configuration namespace, `spark.sql.pipeOperator.*`, for this single feature, @dtenedor ? Do you have a plan to add more subconfigurations ?

Reply:
yea if it's the only one we shouldn't add a new namespace.